eslint.config.js

```diff
@@ -53,4 +54,14 @@ export default [
 			},
 		},
 	},
+
+	// Typescript
+	{
+		files: ["**/*.ts"],
+		...tseslint.configs.recommended[0],
+		rules: {
+			...tseslint.configs.recommended[1].rules,
+			...tseslint.configs.recommended[2].rules,
+		},
+	},
```

Comment:
We should also note that this doesn't fully set up what I'd recommend using typescript-eslint for. There are two more steps to set up:

* Per [https://typescript-eslint.io/getting-started/#next-steps]typescript-eslint > Getting Started > Next Steps], onboarding either or both of:
  * [strict](https://typescript-eslint.io/users/configs#strict): a superset of recommended that includes more opinionated rules which may also catch bugs
  * [stylistic](https://typescript-eslint.io/users/configs#stylistic): additional rules that enforce consistent styling without significantly catching bugs or changing logic
    * Unlike the "stylistic" rules deprecated in ESLint core, the stylistic rules here are actually quite useful for enforcing consistency & using cleaner JS+TS language constructs. We recommend using them.
* Per [typescript-eslint > Typed Linting](https://typescript-eslint.io/getting-started/#typed-linting), enabling lint rules that require type information. 

I think those are reasonable as 2-3 followup tickets, personally.

Reply:
I've added the `strict` & `stylistic` configs, and will setup typed linting in a follow-up PR.